### PR TITLE
Add instructions for building test images

### PR DESCRIPTION
**Pull request by bogdanap**
Attempts to fix: https://github.com/travis-ci/packer-templates/issues/500

Tried to document how we test cookbook changes, based on my (limited) knowledge and after poking around travis-packer-build for a bit.

Please feel free to suggest corrections and improvements. 😁

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -380,3 +380,50 @@ tagset_a = %w(a b c d e f)
...
+and then `bundle exec travis-packer-build [options]`.
+
+Example:
+
+```
+bundle exec travis-packer-build \
+	-I ci-garnet.yml \
+    --target-repo-slug="travis-infrastructure/packer-build" \
+	--github-api-token="<your-token-here>" \
+	--body-tmpl=".packer-build-pull-request-false-tmpl.yml"
```
Comment (meatballhat) [inline README.md]: Indentation inconsistency?

File: README.md [hunk 2]
```diff
@@ -380,3 +380,50 @@ tagset_a = %w(a b c d e f)
...
+example, but you can also create a different template that specifies
+other travis-cookbooks or packer-template branches.
+
+Additionaly, if you just want to test a change in
+[travis-cookbooks](https://github.com/travis-ci/travis-cookbooks), you
+can use the shortcut script in `./bin/packer-build-cookbooks-branch`:
+
+```
+./bin/packer-build-cookbooks-branch <travis-cookbooks-branch-name>
+<template-name>
```
Comment (meatballhat) [inline README.md]: Unintentional line wrap?

File: README.md [hunk 3]
```diff
@@ -155,7 +155,7 @@ cookbooks/travis_ci_connie
 ├── recipes
 │   └── default.rb
 └── spec
-    ├── ...
+	├── ...
```
Comment (meatballhat) [inline README.md]: Unintentional indentation? 😁

Comment (bogdanap) [reply README.md]: That got caught up in my automatic whitespace cleanup, but it didn't look off, so i left it. I can take it out though - just say the word :)

Comment (bogdanap) [reply README.md]: On the other hand, it does look a little off, I'll fix it, sorry about that.

**Comment by edmorley** [comment]
Many thanks!

**Pull request merged**